packages/common/src/components/NbConda.tsx

```diff
@@ -263,25 +265,43 @@ export class NbConda extends React.Component<ICondaEnvProps, ICondaEnvState> {
 
   render(): JSX.Element {
     return (
-      <div className={Style.Panel}>
-        <CondaEnvList
-          height={this.props.height}
-          isPending={this.state.isLoading}
-          environments={this.state.environments}
-          selected={this.state.currentEnvironment}
-          onSelectedChange={this.handleEnvironmentChange}
-          onCreate={this.handleCreateEnvironment}
-          onImport={this.handleImportEnvironment}
-          onRefresh={this.handleRefreshEnvironment}
-          commands={this.props.commands}
-        />
-        <CondaPkgPanel
-          height={this.props.height}
-          width={this.props.width - ENVIRONMENT_PANEL_WIDTH}
-          packageManager={this.props.model.getPackageManager(
-            this.state.currentEnvironment
-          )}
-        />
+      <div className={Style.HeaderContainer}>
+        <div className={Style.Grow}>
+          <div className={Style.Title}>Conda Environments</div>
```

Comment:
Should the title be just Environments?

Reply:
I did debate this a bit myself, I'm happy to go with just environments!